redash/query_runner/__init__.py

```diff
@@ -177,3 +177,19 @@ def get_configuration_schema_for_query_runner_type(query_runner_type):
 def import_query_runners(query_runner_imports):
     for runner_import in query_runner_imports:
         __import__(runner_import)
+
+
+# for athena and presto
```

Comment:
Actually let's move this function into the presto module (since Athena is a product based on Presto) and import it in the athena module.

Also, could you please add a docstring to the function describing what it does please?